cmd/restic/cmd_forget.go

```diff
@@ -129,18 +129,14 @@ func runForget(opts ForgetOptions, gopts GlobalOptions, args []string) error {
 		}
 
 		if policy.Empty() && len(args) == 0 {
-			if !gopts.JSON {
-				Verbosef("no policy was specified, no snapshots will be removed\n")
-			}
+			PrintDef("no policy was specified, no snapshots will be removed\n")
 		}
 
 		if !policy.Empty() {
-			if !gopts.JSON {
-				Verbosef("Applying Policy: %v\n", policy)
-			}
+			PrintDef("Applying Policy: %v\n", policy)
 
 			for k, snapshotGroup := range snapshotGroups {
-				if gopts.Verbose >= 1 && !gopts.JSON {
+				if !gopts.Quiet && !gopts.JSON {
```

Comment:
`gopts.Verbose` != `gopts.verbosity`. `PrintSnapshotGroupHeader` should only be printed if `-v` is passed at least once. However, the check should be `gopts.verbosity >= 2` instead of the current `gopts.Verbose >= 1`.